model/src/main/resources/reference.yml

```diff
@@ -19,6 +19,8 @@
 # exclusive, so this file is only used to show all options and to validate the configuration.
 
 ort:
+  # To overwrite if the reports (analyzer-result.yml) already exists
+  forceOverwrite: true
```

Comment:
@dineshr93 Thank you for the contribution. Please put these lines in alphabetical order (so before `licenseFilePatterns`) and surround it with single empty lines for a better overview. And please also add this line at the same location to the `OrtConfigurationTest.kt` which verifies that the reference.yml is correctly parsed:
`ortConfig.forceOverwrite shouldBe true`

To pass the functional tests you will have to rebase your branch.
To pass the commitlint check you need to change the title to e.g.:
"feat(config): Add forceOverwrite option to reference.yml"

Also, please start the sentence in the commit message body capitalized and end it with a dot, but actually you could just remove it as to me the title is self contained.